src/Features/CSharp/Portable/Completion/CompletionProviders/ObjectCreationCompletionProvider.cs

```diff
@@ -79,6 +79,15 @@ protected override (string displayText, string insertionText) GetDisplayAndInser
                 return (symbol.Name, symbol.Name);
             }
 
+            // If not all  of the type arguments are concrete types, eg
+            // we've inferred something like `Task<T>`, just suggest the
+            // generic type `Task<>`
+            if (symbol is INamedTypeSymbol nt &&
+                nt.TypeArguments.Any(t => t.TypeKind == TypeKind.TypeParameter))
+            {
+                return CompletionUtilities.GetDisplayAndInsertionText(symbol, context);
```

Comment:
Am I supposed to know the difference between `CompletionUtilities.Stuff` and `base.Stuff` intuitively? I see that you're telling me that calling `CompletionUtilities.Stuff` will just do `Task<>` but I don't immediately understand why the difference. Probably just a matter of me not being familiar with this part of the code, but it's somewhat hard to read (I had to go digging around).